### PR TITLE
Update index.html to redirect explore button to resp. page

### PR DESCRIPTION
**Pull request by ishaanbedi**

### CODE REVIEW

**Comment by asdsyd** [comment]
'Closes #6'

**Review by asdsyd — COMMENTED** [review]
For some reason it doesn't work still. take a look.

File: index.html [hunk 1]
```diff
@@ -21,7 +21,9 @@
   <div class="container">
     <div class="left">
       <h1 class="title">WELCOME TO HEALTHY DEVELOPER</h1>
-      <button class="button-81" role="button">Explore</button>
+      <a href="/explore.html">
```
Comment (asdsyd) [inline index.html:24]: https://github.com/asdsyd/Healthy-developer/blob/main/explore.html
Has to be used. Nvm I'll Fix it.
Thank you for contribution ❤️

**Comment by ishaanbedi** [comment]
That's most likely because of your sub hosting structure of URL. Let me change it,

File: index.html (hunk 1 above)
Comment (ishaanbedi) [reply index.html:24]: > https://github.com/asdsyd/Healthy-developer/blob/main/explore.html Has to be used. Nvm I'll Fix it. Thank you for contribution ❤️

Sure! Thank you.

**Pull request merged**